### PR TITLE
adding a title to the tour bubble

### PR DESCRIPTION
**Pull request by andrebautista**
I think it could be extremely useful to have some division of content since most tutorial steps have a title and then a description.  This added an h1 element that can be styled at the users discretion or nothing if the field is left blank.

screen shot 2015-12-09 at 5 49 12 pm

### CODE REVIEW

**Comment by booleanbetrayal** [comment]
We do have template customization support already, but I think this is probably the majority of use-cases as well, so it can go into the default template.

Doing a little house-cleaning. Could you rebase against master real quick and resolve the conflicts? (just rebuild the dist after rebasing). Thanks @andrebautista !

**Comment by andrebautista** [comment]
@booleanbetrayal I just did a `git rebase master` while on my feature branch and I got back "current branch title_element is up to date".

**Comment by booleanbetrayal** [comment]
Make sure you add the main repo as a remote source for your fork. see also
- https://help.github.com/articles/adding-a-remote/

Then you should be able to `git rebase -i upstream/master`

**Comment by andrebautista** [comment]
I thought my rebase would include the last commit where I changed the h2 to h3 but then it just turned into a git mess with I tried to do a git push.

**Comment by andrebautista** [comment]
This took a little more time than I thought due to git troubles but I think this is where you want this PR to be.

File: README.md [hunk 1]
```diff
@@ -118,6 +118,8 @@ There are additional attributes that allow you to customize each tour-tip.
 
 * `tourtip-container-element` **(Default: "body")**: Element the tour tips are placed in. Especially practical when you have several scrollable containers you want to display tours in.
 
+* `tourtip-title` **(Default: null)**: Title creates a separate h2 element to allow the separation of a title and it's content.
```
Comment (booleanbetrayal) [inline README.md]: minor, but documentation indicates h2 but the template is actually h3. mind amending / flattening your commit with that tweak?

**Comment by andrebautista** [comment]
Fixed the readme, should be good to go.

**Comment by booleanbetrayal** [comment]
Thanks!

**Comment by agrigory1982** [comment]
Hello! @andrebautista @booleanbetrayal Usefull update! However I can't install via bower the latest version. I uninstalled the previous version v0.2.5 - 2015-10-02, and install the package again with bower - and still don't have the latest angular-tour.js file downloaded @version v0.2.5 - 2015-12-10

so. i dont have the tourtipTitle available. Could you please, fix it?

**Comment by andrebautista** [comment]
I believe this has something to do with bower trying to install the latest
release not the latest commit to master, this stack overflow post may help.

http://stackoverflow.com/questions/16949173/how-to-install-latest-untagged-state-of-a-repo-using-bower

**Comment by juanvillegas** [comment]
@agrigory1982 yea, it is not released yet so bower doesn't pick it up (as far as bower knows, that is an unreleased work in progress).
Add this to your bower: "angular-tour": "DaftMonk/angular-tour#0dbaea97cbaaa1bd26589e9c7cf664822ba7534a". That will force bower to update angular-tour to the hash after "#", which contains title attribute as well as all other dev work so far, so use at your own risk ;)

**Pull request merged**